### DISCUSSION TITLE
Can this tool change GPU mode when I use USB-C Charging?

### DISCUSSION THREAD

**Discussion by zakimohzani** [seerge/g-helper#288] [Q&A] [answered]
Basically, "optimized" GPU mode doesn't work with USB-C. Can this tool automatically switch to "Standard" when I'm on USB-C? Thank you.

**Comment by seerge** [comment, marked as answer]
Yes, as soon as it gets notification from windows that laptop is plugged and Optimized is selected - it enabled dGPU

**Reply by zakimohzani** [reply]
I just gave it a try and its perfect. It automatically switches on my dGPU with USB-C. 
Its better than the original Armory Crate. Thank you for this invention.
